### PR TITLE
Flexbox

### PR DESCRIPTION
**Pull request by svyatoslavPy**

### CODE REVIEW

**Comment by svyatoslavPy** [comment]
- [DEMO LINK](https://svyatoslavPy.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://svyatoslavPy.github.io/layout_moyo-header/report/html_report/)

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,66 @@
...
+  margin-left: 20px;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  cursor: pointer;
+  color: var(--primary-color);
+}
+
+[data-qa="hover"] {
```
Comment (lerastarynets) [inline src/style.css:49]: pls use only classNames to assign styles

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Pls deploy test reports before rerequesting review. Also the font of your nav looks incorrect pls double check it

File: src/style.css (hunk 1 above)
Comment (svyatoslavPy) [reply src/style.css:53]: add `data-qa="hover"` attribute to the 4th link for testing (`Laptops & computers`)

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,93 @@
...
+  color: var(--primary-color);
+}
```
Comment (anastasiiavorobiova) [inline src/style.css:71]:
```suggestion

```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,93 @@
...
+
+@media screen and (max-width: 1200px) {
+  [data-qa="hover"] {
+    color: var(--secondary-color);
+  }
```
Comment (anastasiiavorobiova) [inline src/style.css:76]:
```suggestion

```
No need to use data attributes for styling. This attribute is created for test purposes only

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Make sure to deploy your page

**Pull request closed without merging**